### PR TITLE
Resolves #2235

### LINKED ISSUE #2235: Incorrect Sample text
This
`PS C:\> Get-AdfsClaimDescription | Where -Object {$_.IsOffered}`
Should Be
`PS C:\> Get-AdfsClaimDescription | Where-Object {$_.IsOffered}`

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bb34b48d-421d-e8fe-a44c-a319c36a5fc7
* Version Independent ID: 376bfa3a-e758-c159-ac1b-5a41c4df3b90
* Content: [Get-AdfsClaimDescription (ADFS)](https://docs.microsoft.com/en-us/powershell/module/adfs/get-adfsclaimdescription?view=win10-ps)
* Content Source: [docset/windows/adfs/Get-AdfsClaimDescription.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/adfs/Get-AdfsClaimDescription.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @Kateyanne
* Microsoft Alias: **v-kaunu**

### PR DESCRIPTION
**Pull request by velkovb**
Fix example 1
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2235

### CODE REVIEW

**Comment by e0i** [comment]
cc @JohanFreelancer9

File: docset/windows/adfs/Get-AdfsClaimDescription.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216)
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Get-AdfsClaimDescription.md:108]:
```suggestion
This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2012r2-ps/adfs/Get-AdfsClaimDescription.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216)
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/adfs/Get-AdfsClaimDescription.md:106]:
```suggestion
This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2012-ps/adfs/Get-ADFSClaimDescription.md [hunk 3]
```diff
@@ -78,7 +74,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/adfs/Get-ADFSClaimDescription.md:77]:
```suggestion
This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**